src/java.base/share/classes/jdk/internal/misc/OperatingSystemProps.java.template

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package jdk.internal.misc;
+
+/**
+ * The corresponding source file is generated by GensrcMisc.gmk for java.base.
+ * @see OperatingSystem
+ */
+class OperatingSystemProps {
+
+    // Unique integers named to match the build system naming of the build target
+    // The values must match the ordinals of the respective enum
+    private static final int TARGET_OS_linux   = 0;
+    private static final int TARGET_OS_macosx  = 1;
```

Comment:
`TARGET_OS_macos` maybe?

Reply:
These names need to match the names used in the build makefiles; currently is macosx.